### PR TITLE
Add links to Payment policy, Privacy policy, Terms

### PR DESCRIPTION
**Pull request by robojack**
The "add to calendar" button is also hidden

### CODE REVIEW

File: app/screens/GetStartedScreen.js [hunk 1]
```diff
@@ -57,9 +57,23 @@ const GetStartedScreen = ({navigation}) => {
...
       <View style={{flex: 1}} />
 
-      <TouchableOpacity style={{padding: 20}}>
-        <TiText style={styles.legalText}>LEGAL DISCLOSURE</TiText>
-      </TouchableOpacity>
+      <TiText style={[styles.label, {textAlign: 'center'}]}>
+        <Text
+          style={styles.labelEmphasis}
+          onPress={() =>
+            Linking.openURL('https://www.theinformation.com/terms')
```
Comment (bolandrm) [inline app/screens/GetStartedScreen.js]: could we make these URLs constants, perhaps in `Config`?

File: app/components/CalendarList/CalendarListItem.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ const CalendarListItem = ({item}) => {
           </Text>
         )}
       </TiText>
-      <Button text="ADD TO MY CALENDAR" fullWidth={true} />
+      {/* <Button text="ADD TO MY CALENDAR" fullWidth={true} /> */}
```
Comment (bolandrm) [inline app/components/CalendarList/CalendarListItem.js]: were you going to hide recommend/share in this pr as well?

File: app/screens/GetStartedScreen.js (hunk 1 above)
Comment (robojack) [reply app/screens/GetStartedScreen.js]: good idea 👍

File: app/components/CalendarList/CalendarListItem.js (hunk 2 above)
Comment (robojack) [reply app/components/CalendarList/CalendarListItem.js]: This will be included as well

**Pull request merged**